File: gmail_smtp.py
#run with argument passing
#Made by Vionikk

import smtplib
import sys
import datetime
from email.mime.multipart import MIMEMultipart
from email.mime.text import MIMEText

if(len(sys.argv)!=1):
    email=sys.argv[1]
    subject=sys.argv[2] 

    credentials = ["yourgmailaddress@gmail.com","yourpassword"]

    smtpObj = smtplib.SMTP('smtp.gmail.com', 587) #you must allow it: https://myaccount.google.com/lesssecureapps
    smtpObj.starttls()
    smtpObj.login(credentials[0],credentials[1])


    msg = MIMEMultipart()
    msg['From'] = credentials[0]
    msg['To'] = email
    msg['Subject'] = subject
    
    body = f"{datetime.datetime.now()} Hello, gmail;)"
    msg.attach(MIMEText(body, 'plain'))




    smtpObj.sendmail(credentials[0],email,msg.as_string())
    smtpObj.quit()
else:
    print("Help: \n python script4.py [ToEmail] [Subject]")

File: gmail_smtp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Net;
using System.Net.Mail;
using static System.Console;

//Made by Vionikk
/*
  run with argument passing
*/
namespace Lab1._3
{
    class Program
    {
        static void Main(string[] args)
        {
            if (args.Length == 2)
            {
                try
                {
                    SmtpClient CL = new SmtpClient("smtp.gmail.com"); //you must allow it: https//myaccount.google.com/lesssecureapps
                    CL.Port = 587;
                    CL.Credentials = new NetworkCredential("yourgmailaddress@gmail.com", "yourpassword"); //input your gmail address and password
                    CL.EnableSsl = true;
                    CL.DeliveryMethod = SmtpDeliveryMethod.Network;
                    CL.Send(new MailMessage("yourgmailaddress@gmail.com", args[0], args[1], $"{DateTime.Now} Hello, gmail;)"));
                    Console.WriteLine("Message was send...");
                }
                catch (Exception e)
                {
                    WriteLine(e.Message);
                }
            }
            else
            {
                Console.WriteLine("Incorrect usage!");
                Console.WriteLine("Two arguments required: <E-mail address> <Subject>");
            }
        }
    }
}
